README.md

```diff
@@ -41,7 +41,7 @@ It builds the following ROM:
 
 ### Windows
 
-For Windows 10, install WSL and a distribution by following this
+For Windows 10, install WSL and distribution by following this
```

Comment:
`a distribution` is correct. A user can install any compatible Linux distribution along with WSL.